modules/i18n/src/translations/en-US/portals/branding.ts

```diff
@@ -168,6 +156,48 @@ export const branding: BrandingNS = {
         "sms-otp": "SMS OTP",
         "totp": "TOTP"
     },
+    ai: {
+        banner: {
+            full: {
+                heading: "Simplify branding setup with Branding AI",
+                subHeading: "Elevate your organization's branding effortlessly with BrandingAI, simply by providing your website URL.",
+                button: "Try Branding AI"
+            },
+            input: {
+                heading: "Configure branding with Branding AI",
+                subHeading: "Enter your organization's website URL; our AI efficiently customizes branding elements to match your organization.",
+                placeholder: "Enter website URL",
+                button: "Generate Branding"
+            },
+            collapsed: {
+                heading: "Configure branding with Branding AI",
+                subHeading: "Enter your organization's website URL; our AI efficiently customizes branding elements to match your organization.",
+                button: "Try Branding AI"
+            }
+        },
+        screens: {
+            loading: {
+                heading: "Generating your branding",
+                didYouKnow: "Did you know?",
+                facts: {
+                    0: "Asgardeo's advanced theming capabilities let you customize your site title, copyright info, and support email on login pages to match your brand identity",
```

Comment:
Let's pass productName through i18n options and placeholders.

```suggestion
                    0: "{{productName}}'s advanced theming capabilities let you customize your site title, copyright info, and support email on login pages to match your brand identity",
```